src/agent/rustjail/src/cgroups/systemd/subsystem/cpu.rs

```diff
@@ -81,7 +84,9 @@ impl Cpu {
     ) -> Result<()> {
         if let Some(shares) = cpu_resources.shares {
             let weight = shares_to_weight(shares);
-            properties.push(("CPUWeight", Value::U64(weight)));
```

Comment:
Could you add a reference to https://github.com/systemd/systemd/blob/d19434fbf81db04d03c8cffa87821f754a86635b/src/basic/cgroup-util.h#L91 like the one for CPUShares? Thanks!

Reply:
Thanks @bergwolf -, fixed.